packages/renderer/src/stores/all-installed-extensions.ts

```diff
@@ -0,0 +1,60 @@
+/**********************************************************************
+ * Copyright (C) 2024 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+// Combine PD and DD extensions being installed
+import { derived, type Readable } from 'svelte/store';
+import { extensionInfos } from './extensions';
+import { contributions } from './contribs';
+import type { ExtensionInfo } from '../../../main/src/plugin/api/extension-info';
+
+export interface CombinedExtensionInfoUI extends ExtensionInfo {
+  type: 'dd' | 'pd';
```

Comment:
Why using short names ? Feel a bit confusing to read

Reply:
to avoid any trademark issue

Reply:
I see, even adding a comment is risky ?

Reply:
I'll add the comment, it's OK

Reply:
@axel7083 comment added, does it work for you ?

Reply:
Yes perfect!! Thanks for it